### ISSUE TITLE
Preenchimento do Select, Textarea e Datapiker no Update

### ISSUE THREAD

**Issue by imbitubadigital** [Rocketseat/unform#47]
Doing my first test, i realize  that the **Select** (simple e multiple) aren't marked with the value for  **initialData** in the update moment. Like how the values of **initialData** are not passed on **Textearea** and **Datapiker** loading the form for update. Any sugestion for how to fix it?

**Comment by leonardobrito** [comment]
Hi @imbitubadigital , we want to keep all issues and contributions to this repo in english so can you please translate the original message here? So that more people can participate in solving the problem. (:

**Comment by diego3g** [comment]
@imbitubadigital You should display the form with initialData only after loading that data, so, as soon as the loading is true do not mount the form inside the component. Your code would look like this:
```js
render() {
  return loading ? <Loading /> : <Form initialData={data}>...</Form>
}
```
TLDR: Do not render form before loading the initialData.

I'll close the issue for now but feel free to reopen it if you face any issues.
